.mergify.yml

```diff
@@ -1,10 +1,49 @@
+# This file was automatically generated by sbt-typelevel-mergify using the
+# mergifyGenerate task. You should add and commit this file to
+# your git repository. It goes without saying that you shouldn't edit
+# this file by hand! Instead, if you wish to make changes, you should
+# change your sbt build configuration to revise the mergify configuration
+# to meet your needs, then regenerate this file.
 
 pull_request_rules:
-  - name: automatically merge scala-steward's PRs
-    conditions:
-      - author=scala-steward
-      - body~=labels:.*semver-patch.*
-      - status-success=build
-    actions:
-      merge:
-        method: merge
+- name: merge scala-steward's PRs
+  conditions:
+  - author=scala-steward
+  - or:
+    - body~=labels:.*early-semver-patch
+    - body~=labels:.*early-semver-minor
+  - status-success=Build and Test (ubuntu-latest, 2.12.15, temurin@17)
+  actions:
+    merge: {}
+- name: Label app PRs
+  conditions:
+  - files~=^app/
+  actions:
+    label:
+      add:
+      - app
+      remove: []
+- name: Label core PRs
+  conditions:
+  - files~=^core/
+  actions:
+    label:
+      add:
+      - core
+      remove: []
```

Comment:
Do you care about this feature? It labels PRs based on which sbt module the file changes are made. Otherwise I can disable it.